beacon_node/client/src/builder.rs

```diff
@@ -70,6 +71,7 @@ pub struct ClientBuilder<T: BeaconChainTypes> {
     gossipsub_registry: Option<Registry>,
     db_path: Option<PathBuf>,
     freezer_db_path: Option<PathBuf>,
+    blobs_db_path: Option<PathBuf>,
```

Comment:
I couldn't find where this field is being accessed, looks like we may not need it?

Reply:
Yeah I agree. We should be setting the blob DB path via `disk_store` here: https://github.com/sigp/lighthouse/blob/5c5afafc0dd7f499c8855634db53f8ae664f159d/beacon_node/client/src/builder.rs#L950-L958

Reply:
Yeah I agree. We should be setting the blob DB path via `disk_store` here: https://github.com/sigp/lighthouse/blob/5c5afafc0dd7f499c8855634db53f8ae664f159d/beacon_node/client/src/builder.rs#L950-L958

Reply:
updated here b9433a5720a7a686b9899d3c96caffdc0e28624d